project-reports/2024/2024-annual-Hyperledger-Cello.md

```diff
@@ -0,0 +1,59 @@
+---
+layout: default
+title: 2024 Annual Review Hyperledger Fabric
+parent: 2024
+grand_parent: Project Updates
+---
+
+# Project Health
+
+Hyperledger Cello continues to evolve as an important project within the Hyperledger community, focusing on providing the solution to operate the blockchain platform such as Hyperledger Fabric.
+Over the past year, the project has seen consistent development efforts, with contributions from a diverse set of individuals and organizations.
+The first major release of [v1.0.0](https://github.com/hyperledger/cello/releases/tag/1.0.0) is done on November, 2023.
+Most activities happened at Github, including discussions, code contributions, bug reports.
+[Weekly meetings](https://wiki.hyperledger.org/display/cello/Weekly+Meeting) are record at Hyperledger Wiki page.
+
+[LFX Insights report for 2023](https://insights.lfx.linuxfoundation.org/foundation/hyp/overview?project=cello&bestPractice=false&repository=&dateFilters=Last%20Year&dateRange=2023-01-01%20to%202023-12-31&compare=PP&granularity=month&hideBots=true)
+
+Highlights from the 2023 Insights report:
+- 20 contributors (-20% from 2022)
+- 107 commits (-41% from 2022)
+- 63% of code activities were performed by 3 people (Oracle, H3c and other)
+- 62% of code activities were performed by 2 organization (Oracle and H3c)
+- Total of 10 organizations contributed, which indicates that there are many contributors and organizations with small contributions
+
+# Maintainer Diversity
+
+There are 4 maintainers from different organizations, indicating a good diversity:
+
+There are four new contributors from the intern program, and they start to make small contributions as well as learning necessary knowledge and tools
+
+[Maintainers List](https://github.com/hyperledger/cello/blob/main/MAINTAINERS.md)
+
+# Project Adoption
+
+We do not have a statistic of the adoption, but based on the github issues and discussions, there are adoptions to use Cello as Blockchain management platform.
```

Comment:
@yeasy One thing that I've seen work well in the past is to open an issue in the GitHub issue tracker with the title "Who uses $PROJECT_NAME?" and then in there explain that you are collecting some information about users and their use-cases (as much or as little as they can/willing to volunteer).

Not a requirement nor a change request in the report, just an idea in case you were interested.

Reply:
Good idea, we could track using the wiki page. Create one at https://github.com/hyperledger/cello/wiki/Who-Uses-the-Project.

Reply:
@yeasy please update the report with this